configs/core/config.txt.armhf

```diff
@@ -0,0 +1,19 @@
+[pi4]
```

Comment:
I like that you still kept the config.txt from core intact, just cleaned it up properly. It's good to keep the non-critical changes to minimum, not to risk regressing too much.